corehq/ex-submodules/casexml/apps/case/util.py

```diff
@@ -186,16 +186,25 @@ def iter_cases(case_ids, strip_history=False, wrap=True):
 def property_changed_in_action(case_transaction, case_id, case_property_name):
     from casexml.apps.case.xform import get_case_updates
     PropertyChangedInfo = namedtuple("PropertyChangedInfo", 'transaction new_value modified_on')
-    update_actions = [
-        (update.modified_on_str, update.get_update_action(), case_transaction)
-        for update in get_case_updates(case_transaction.form)
-        if update.id == case_id
-    ]
-    for (modified_on, update_action, case_transaction) in update_actions:
-        if update_action:
-            property_changed = update_action.dynamic_properties.get(case_property_name)
+    include_create_fields = case_property_name in ['owner_id', 'name', 'external_id']
+    case_updates = get_case_updates(case_transaction.form)
+
+    actions = []
+    for update in case_updates:
+        if update.id == case_id:
+            actions.append((update.modified_on_str, update.get_update_action(), case_transaction))
+            if include_create_fields:
+                actions.append((update.modified_on_str, update.get_create_action(), case_transaction))
```

Comment:
Is it valid to change any of the `include_create_fields` after it's already been created? the particular scenario I'm thinking of is reassigning cases would supply the owner_id in an update action and then with this change also include a second(third,fourth...) create action

Reply:
good point.